source/extensions/filters/http/common/jwks_fetcher.cc

```diff
@@ -38,6 +76,7 @@ class JwksFetcherImpl : public JwksFetcher,
     complete_ = false;
     receiver_ = &receiver;
     uri_ = &uri;
```

Comment:
I am debating if we should remove "uri" in fetch() call.  "uri" is part of "remote_jwks".    Since you pass in "remote_jwks" in the constructor,  we don't need "uri" here.  If we remove it,  it make Jwks_Fetcher cleaner,  it just takes "remote_jwks" as config input.

Reply:
also implemented.